.eslintrc.json

```diff
@@ -42,8 +42,7 @@
       "extends": ["plugin:prettier/recommended"],
       "rules": {
         // NOTE: WE ARE OVERRIDING THE DEFAULT CONFIG TO ALWAYS SET THE PARSER TO ANGULAR (SEE BELOW)
-        "prettier/prettier": ["error", { "parser": "angular" }],
-        "@typescript-eslint/no-explicit-any": "off"
+        "prettier/prettier": ["error", { "parser": "angular" }]
```

Comment:
**question**
Do we still need this rule?

Reply:
Also, let's add new tech task in backlog to switch to the latest version of eslint config